src/components/app/app.js

```diff
@@ -38,6 +38,9 @@ class App extends React.Component {
     if (prevProps.data !== this.props.data) {
       this.updatePipelineData();
     }
+    if (prevProps.options !== this.props.options) {
```

Comment:
As per my understanding this checks for reference and not the actual values inside the object. I wonder how this works when a value inside the object changes. Are we always creating a new object when any value inside the object changes ?

Reply:
@ravi-kumar-pilla As `options ` is object and any change in parent component, will make `prevProps.options !== this.props.options` as always true. So to make sure we look for correct value change inside object, Now I use `isEqual` from lodash for `options` object comparison.